### PR TITLE
Align `CONTAINS` et al with comparability

### PR DESCRIPTION
**Pull request by Mats-SX**
With the introduction of the thorough definition of comparability, we need to sort these operators in line to achieve consistency.

### CODE REVIEW

File: cip/2.testable/CIP2015-09-15-STARTS-WITH-and-ENDS-WITH.adoc [hunk 1]
```diff
@@ -78,6 +82,12 @@ contains      = expression, “CONTAINS” expression ;
 
 === Semantics
 
+All three proposed operators are defined as _comparison operators_ on string operands.
+As such, behavioural semantics need to be in line with what is detailed by the link:https://github.com/opencypher/openCypher/blob/master/cip/1.accepted/CIP2016-06-14-Define-comparability-and-equality-as-well-as-orderability-and-equivalence.adoc[Comparability CIP].
+This has the following consequence:
+
+- If any argument to `STARTS WITH`, `ENDS WITH`, or `CONTAINS` is `null` or a non-string, then the result of evaluating the whole predicate is `null`.
```
Comment (petraselmer) [inline cip/2.testable/CIP2015-09-15-STARTS-WITH-and-ENDS-WITH.adoc]: "or a non-string,"
->
"or not a string,"

**Comment by petraselmer** [comment]
@Mats-SX looks good! Just one small comment, and then it's ready to merge

**Comment by Mats-SX** [comment]
This should include updates to the TCK.

**Comment by Mats-SX** [comment]
@petraselmer This may now be merged.

**Pull request merged**